docs/resources/environment.md

```diff
@@ -0,0 +1,33 @@
+# Aptible Environment Resource
+
+This resource is used to create and manage [Environments](https://deploy-docs.aptible.com/docs/environments) running on Aptible Deploy.
+
+## Example Usage
+
+```hcl
+resource "aptible_environment" "example_environment" {
+  handle = "example"
+  stack_id = 1
+  org_id = "123e4567-e89b-12d3-a456-42661417400"  # insert your org uuid here!
+}
+```
+
+## Argument Reference
+
+- `stack_id` (Required) - The id of the [stack](https://www.aptible.com/documentation/deploy/reference/stacks.html) you would like the environment to be provisioned on.  
```

Comment:
www.aptible.com/documentation is the old docs site. deploy-docs.aptible.com is the current one.

```suggestion
- `stack_id` (Required) - The id of the [stack](https://deploy-docs.aptible.com/docs/stacks) you would like the environment to be provisioned on.  
```

Reply:
verified link, lgtm